src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/CrazyPriceHost.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Exadel.CrazyPrice.Common.Configurations;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.IO;
+
+namespace Exadel.CrazyPrice.Common
+{
+    public static class CrazyPriceHost
+    {
+        public static void CheckLoggerAndStartHost(IHostBuilder hostBuilder,
+            string initMessage, string errorInitMessage)
+        {
+            const string useLogging = "Logging:UseLogging";
```

Comment:
No sense to create constants here - put them to class or just put in code.

Reply:
Removed.